fem/bilininteg.hpp

```diff
@@ -2995,6 +2997,26 @@ class ElasticityIntegrator : public BilinearFormIntegrator
    Vector divshape;
 #endif
 
+   // PA extension
+   std::shared_ptr<QuadratureFunction> lambda_quad, mu_quad;
+   std::shared_ptr<QuadratureFunction> q_vec;
+   std::shared_ptr<QuadratureSpace> quad_space;
+
+   const DofToQuad *maps;         ///< Not owned
+   const GeometricFactors *geom;  ///< Not owned
+   int vdim, ndofs;
+   const FiniteElementSpace
+   *fespace;   ///< Not owned. Not const because it is used in a getter to construct bilinearforms which require non-const fespaces for some reason. Can it be const?
```

Comment:
Formatting (make variable definition one line).

Is the comment out of date? It is const…

Reply:
It was out of date, I just fixed it.